internal/compiler/parser/document.rs

```diff
@@ -163,6 +164,24 @@ fn parse_export(p: &mut impl Parser) -> bool {
         }
     } else if p.peek().as_str() == "struct" {
         parse_struct_declaration(&mut *p)
+    } else if p.peek().as_str() == "*" {
+        let mut p = p.start_node(SyntaxKind::ExportModule);
+        p.consume(); // *
```

Comment:
could be an `expect()`

Reply:
I've changed it to use `SyntaxKind::Star` instead.